tests/test_pytest_plugin.py

```diff
@@ -243,3 +246,13 @@ def test_finalized() -> None:
     testdir.makeconftest(CONFTEST)
     result = testdir.runpytest('-p', 'no:sugar', '--aiohttp-loop=pyloop')
     result.assert_outcomes(passed=2)
+
+
+def test_warnings_propagated():
+    with warnings.catch_warnings(record=True) as log:
```

Comment:
You should use a built-in `recwarn` fixture for testing warnings in pytest: https://docs.pytest.org/en/latest/warnings.html#recwarn.

Reply:
Fixed, thanks!